webapp/tests/karma/unit/services/format-data-record.js

```diff
@@ -76,7 +76,7 @@ describe('FormatDataRecord service', () => {
         chai.expect(row.messages.length).to.equal(1);
         const message = row.messages[0];
         chai.expect(message.to).to.equal('+123456');
-        chai.expect(message.error).to.equal('messages.errors.message.empty');
+        //chai.expect(message.error).to.equal('messages.errors.message.empty');
```

Comment:
Is this commented out by mistake? This test seems to have nothing to do with the code updated by this PR.